graylog2-web-interface/src/views/components/DashboardSearchBar.test.tsx

```diff
@@ -73,8 +79,6 @@ describe('DashboardSearchBar', () => {
 
     fireEvent.click(searchButton);
 
-    await waitFor(() => expect(onExecute).toHaveBeenCalledTimes(1));
-
-    expect(GlobalOverrideActions.set).toHaveBeenCalledWith(undefined, '');
```

Comment:
Should we add another tests that is changing the override and executing the search? If not, can we remove the `GlobalOverrideStore` mock?

Reply:
I would have loved to, but we can't execute a submission as it seems:
https://github.com/jsdom/jsdom/issues/1937